ibr/ibr-sdk/build/ibr-sdk.js

```diff
@@ -6,6 +6,8 @@
 
 import {OrbitControls} from
   './../../../node_modules/three/examples/jsm/controls/OrbitControls.js';
+import {IBRObject} from
+  './../class/IBRObject.js';
```

Comment:
we need to eventually move away from relative imports, it's not scalable as a production library